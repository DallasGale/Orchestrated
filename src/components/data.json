{
  "games": [
    {
      "id": 0,
      "img": "",
      "title": "Cyberpunk 2077",
      "description": "A futuristic first person shooter cross ARPG set in the dystopian metropolis of Night City.",
      "cost": {
        "now": 80,
        "was": 99
      },
      "price": 80,
      "sale": true,
      "inCart": false,
      "units": 1,
      "showRemove": false
    },
    {
      "id": 1,
      "img": "",
      "title": "Star Wars Jedi: Fallen Order",
      "description": "A third person ARPG set in the Star Wars universe where you play as a young Jedi.",
      "cost": {
        "now": 70,
        "was": 99
      },
      "price": 70,
      "sale": true,
      "inCart": false,
      "units": 1,
      "showRemove": false
    },
    {
      "id": 2,
      "img": "",
      "title": "The Last of Us Part II",
      "description": "A story based action survival game based in a post apocolyptic workd.",
      "cost": {
        "now": 91,
        "was": 105
      },
      "price": 91,
      "sale": true,
      "inCart": false,
      "units": 1,
      "showRemove": false
    }
  ]
}