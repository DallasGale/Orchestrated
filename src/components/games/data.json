{
  "games": [
    {
      "id": 0,
      "img": "",
      "title": "Cyberpunk 2077",
      "description": "A futuristic first person shooter cross ARPG set in the dystopian metropolis of Night City.",
      "price": {
        "now": 80,
        "was": 99
      },
      "sale": true
    },
    {
      "id": 1,
      "img": "",
      "title": "Star Wars Jedi: Fallen Order",
      "description": "A third person ARPG set in the Star Wars universe where you play as a young Jedi.",
      "price": {
        "now": 70,
        "was": 99
      },
      "sale": true
    },
    {
      "id": 2,
      "img": "",
      "title": "The Last of Us Part II",
      "description": "A story based action survival game based in a post apocolyptic workd.",
      "price": {
        "now": 91,
        "was": 105
      },
      "sale": true
    }
  ]
}