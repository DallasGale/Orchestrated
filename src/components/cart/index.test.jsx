import React from 'react'
import { render } from 'enzyme'
import toJson from 'enzyme-to-json'
import Cart from '.'

import data from '../data.json'

const props = {
  cart: data.games,
}

describe('<Cart />', () => {
  test('it renders', () => {
    const Component = render(<Cart {...props} />)
    expect(toJson(Component)).toMatchSnapshot()
  })
})
