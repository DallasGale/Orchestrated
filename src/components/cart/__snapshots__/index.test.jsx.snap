// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[` 1`] = `
<div
  class="sc-EHOje jGEDmf"
>
  <div
    class="sc-bdVaJa dqmyKm"
  >
    <h3
      class="sc-bxivhb XQmtH"
    >
      Your Cart
    </h3>
  </div>
  <div
    class="sc-bZQynM edqsDC"
  >
    <div>
      <div
        class="utils__padding-l--20"
      >
        Cyberpunk 2077
      </div>
    </div>
    <small
      class="typography__align--right"
    >
      Amount
    </small>
    <small
      class="typography__align--center sc-gzVnrw jAwSlX"
    >
      1
    </small>
    <div
      class="typography__align--right"
    >
      <div
        class="utils__padding-r--20"
      >
        $80
      </div>
    </div>
  </div>
  <div
    class="sc-bZQynM edqsDC"
  >
    <div>
      <div
        class="utils__padding-l--20"
      >
        Star Wars Jedi: Fallen Order
      </div>
    </div>
    <small
      class="typography__align--right"
    >
      Amount
    </small>
    <small
      class="typography__align--center sc-gzVnrw jAwSlX"
    >
      1
    </small>
    <div
      class="typography__align--right"
    >
      <div
        class="utils__padding-r--20"
      >
        $70
      </div>
    </div>
  </div>
  <div
    class="sc-bZQynM edqsDC"
  >
    <div>
      <div
        class="utils__padding-l--20"
      >
        The Last of Us Part II
      </div>
    </div>
    <small
      class="typography__align--right"
    >
      Amount
    </small>
    <small
      class="typography__align--center sc-gzVnrw jAwSlX"
    >
      1
    </small>
    <div
      class="typography__align--right"
    >
      <div
        class="utils__padding-r--20"
      >
        $91
      </div>
    </div>
  </div>
  <div
    class="sc-bdVaJa dqmyKm"
  >
    <h4
      class="typography__align--right sc-ifAKCX hkxIZy"
    >
      TOTAL: $undefined
    </h4>
  </div>
</div>
`;

exports[`<Cart /> it renders 1`] = `
<div
  class="sc-EHOje jGEDmf"
>
  <div
    class="sc-bdVaJa dqmyKm"
  >
    <h3
      class="sc-bxivhb XQmtH"
    >
      Your Cart
    </h3>
  </div>
  <div
    class="sc-bZQynM edqsDC"
  >
    <div>
      <div
        class="utils__padding-l--20"
      >
        Cyberpunk 2077
      </div>
    </div>
    <small
      class="typography__align--right"
    >
      Amount
    </small>
    <small
      class="typography__align--center sc-gzVnrw jAwSlX"
    >
      1
    </small>
    <div
      class="typography__align--right"
    >
      <div
        class="utils__padding-r--20"
      >
        $80
      </div>
    </div>
  </div>
  <div
    class="sc-bZQynM edqsDC"
  >
    <div>
      <div
        class="utils__padding-l--20"
      >
        Star Wars Jedi: Fallen Order
      </div>
    </div>
    <small
      class="typography__align--right"
    >
      Amount
    </small>
    <small
      class="typography__align--center sc-gzVnrw jAwSlX"
    >
      1
    </small>
    <div
      class="typography__align--right"
    >
      <div
        class="utils__padding-r--20"
      >
        $70
      </div>
    </div>
  </div>
  <div
    class="sc-bZQynM edqsDC"
  >
    <div>
      <div
        class="utils__padding-l--20"
      >
        The Last of Us Part II
      </div>
    </div>
    <small
      class="typography__align--right"
    >
      Amount
    </small>
    <small
      class="typography__align--center sc-gzVnrw jAwSlX"
    >
      1
    </small>
    <div
      class="typography__align--right"
    >
      <div
        class="utils__padding-r--20"
      >
        $91
      </div>
    </div>
  </div>
  <div
    class="sc-bdVaJa dqmyKm"
  >
    <h4
      class="typography__align--right sc-ifAKCX hkxIZy"
    >
      TOTAL: $undefined
    </h4>
  </div>
</div>
`;
